src/main.rs

```diff
@@ -89,21 +91,20 @@ fn main() {
         None
     };
 
-    let tpu_client_pool = Arc::new(RotatingQueue::<Arc<TpuClient<QuicPool, QuicConnectionManager, QuicConfig>>>::new(
-        number_of_tpu_clients,
-        || {
-            let quic_connection_cache = quic_connection_cache.clone();
-            Arc::new(
-                TpuClient::new_with_connection_cache(
-                    rpc_clients.get().clone(),
-                    &websocket_url,
-                    solana_client::tpu_client::TpuClientConfig::default(),
-                    quic_connection_cache.unwrap(),
-                )
-                .unwrap(),
+    let tpu_client_pool = Arc::new(RotatingQueue::<
+        Arc<TpuClient<QuicPool, QuicConnectionManager, QuicConfig>>,
```

Comment:
not related to this PR, but I don't think there is a value in having many TpuClients since each of them internally sends to the next N leaders (4 i think)

Reply:
Yes i think I have removed it in my other PR right.